backend/src/test/java/gov/cdc/usds/simplereport/utils/TestResultBulkUploadToFhirTest.java

```diff
@@ -44,16 +44,17 @@
 import org.springframework.boot.info.GitProperties;
 
 @ExtendWith(MockitoExtension.class)
-public class BulkUploadResultsToFhirTest {
+public class TestResultBulkUploadToFhirTest {
```

Comment:
Renaming just to make prevent mistaking bulk upload cases between test result and condition agnostic